### ISSUE TITLE
Expose logs feature via terraform provider

### ISSUE THREAD

**Issue by velkovb-nuk** [newrelic/terraform-provider-newrelic#1043]
### Feature Description
Want to be able to manage the `Logs` configuration via terraform. Log views, drop filters, parsing

### Describe Alternatives
The alternative for now is doing it from the NewRelic UI.

### Additional context
Add any other context here.

**Comment by fr-ser** [comment]
At least drop filters seem to be supported. Right?
https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/nrql_drop_rule

**Comment by sqills-jerh** [comment]
It would also be amazing if we could add the created log views to a dashboard via terraform.

**Comment by oliver-hohn** [comment]
Support around creating syslog drains would also be a nice to have: https://docs.newrelic.com/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/heroku-log-forwarding/#register-syslog.

Although, this might be tricky to support. There doesn't seem to be a public API for configuring log integrations: https://docs.newrelic.com/docs/apis/intro-apis/introduction-new-relic-apis/.

**Comment by aelmekeev** [comment]
With the migration of our logs to NewRelic we would like to use a nice feature of saved views as the way to onboard team members on what logs are exposed and what are the most useful fields for each type.

Unfortunately, it's a bit tricky to manage those views without terraform since we have quite a lot of account and log types.

So it would be great to see log views managed in terraform!

**Comment by ruwanvm** [comment]
Is this feature implemented? If not can we see the progress here

**Comment by advissor** [comment]
We are missing this feature :) 
Any ETA on this?

**Comment by AlexeyDP** [comment]
Nice feature, waiting on it as it would be really helpful

**Comment by kidk** [comment]
We're currently researching the work. Will give you all an update when we know more.

**Comment by whyman10x** [comment]
Its been another 3 months... any news?

**Comment by sqills-jerh** [comment]
> Its been another 3 months... any news?

Afaik, there's a new log table widget:
https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/one_dashboard#widget_log_table

**Comment by kidk** [comment]
Parsing and obfuscation have recently been added with data partitions being released soon. If you feel any other features are missing I'd like to ask to create a specific issue for them. I'm going to close this ticket, but feel free to continue the conversation.
